src/Microsoft.AspNetCore.Razor.LanguageServer.Common/Serialization/TagHelperDescriptorJsonConverter.cs

```diff
@@ -133,7 +133,7 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
             writer.WriteEndObject();
```

Comment:
All the changes here are a direct copy from Razor. Sadly our serialization logic is in the VS windows specific binary.